src/app/views/user/user-settings/user-profile/user-profile.component.html

```diff
@@ -1,82 +1,122 @@
+        <p>{{'user.profile.form.permit.upload.description' | translate}}</p>
+        <div *ngIf="user.permitImage" class="permit-saved-message">{{'user.profile.form.permit.upload.uploaded' | translate}}</div>
+      </div>
 
-      <label for="work-experience">{{'profile.form.experience' | translate | uppercase}}</label>
-      <textarea name="workExperience" id="work-experience" class="material-input" [disabled]="!editMode" [(ngModel)]="user.workExperience"></textarea>
-    </div>
+      <hr>
 
-    <div class="material-group">
-      <i class="icon-ic-education"></i>
+      <div class="profile-input-container">
+        <label>{{'user.profile.form.languages.label' | translate}}:</label>
+        <autocomplete-dropdown (dropdownListItemSelect)="onLanguageSelect($event)" [clearDestinationAfterSelection]="true" [placeholder]="'user.profile.form.languages.placeholder' | translate" [autocompleteResultLabelFunction]="namePropertyLabel" [selectedItemLabelFunction]="namePropertyLabel"
+           [getData]="getLanguages()">
+        </autocomplete-dropdown>
 
-      <label for="education">{{'profile.form.education' | translate | uppercase}}</label>
-      <textarea name="education" id="education" class="material-input" [disabled]="!editMode" [(ngModel)]="user.education"></textarea>
-    </div>
+        <div class="profile-input-languages-table-container">
+          <div class="profile-input-languages-table-row-container" *ngFor="let userLanguage of user.userLanguages">
+            <div class="profile-input-languages-table-row-inner-container" *ngIf="userLanguage.proficiency.proficiency !== 5">
+              <div class="profile-input-languages-table-row-inner-language-container"><p>{{userLanguage.language.name}}</p></div>
+              <div class="profile-input-languages-table-row-inner-icon-container" (click)="onRemoveUserLanguage(userLanguage)">
+                <i class="fa fa-times" aria-hidden="true"></i>
+              </div>
+              <div class="profile-input-languages-table-row-inner-dropdown-container">
+                <autocomplete-dropdown [lookupType]="'list'" [enumList]="languageProficiencyLevelsAvailable" [(destination)]="userLanguage.proficiency" [placeholder]="'language.proficiency.placeholder' | translate" [autocompleteResultLabelFunction]="namePropertyLabel"
+                    [selectedItemLabelFunction]="namePropertyLabel">
+                </autocomplete-dropdown>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
 
-    <div class="material-group">
-      <i class="icon-ic-tool"></i>
+      <div class="profile-input-container">
+        <label>{{'user.profile.form.skills.label' | translate}}:</label>
+        <textarea class="form-control material-input" name="skills" id="skills" class="form-control material-input" autosize type="text" placeholder="{{'user.profile.form.skills.placeholder' | translate}}" [(ngModel)]="user.skills"></textarea>
```

Comment:
Or perhaps you want it changed to skills dropdown? Is the skills stuff fully implemented in the API?